erpnext/accounts/report/trial_balance/trial_balance.py

```diff
@@ -87,8 +87,7 @@ def get_opening_balances(filters):
 
 
 def get_rootwise_opening_balances(filters, report_type):
-	additional_conditions = " and posting_date >= %(year_start_date)s" \
-		if report_type == "Profit and Loss" else ""
+	additional_conditions = ""
```

Comment:
i think considering the trial balance sheet has the period closing entry check is it not more confusing if you have only have half the accounts showing opening balance and half not showing